core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostServiceImpl.scala

```diff
@@ -116,34 +212,6 @@ class SubmissionCostServiceImpl(defaultTableName: String,
     s"""AND $datePartitionColumn BETWEEN "$windowStartDate" AND "$windowEndDate""""
   }
 
-  private def executeSubmissionCostsQuery(submissionId: String,
-                                          googleProjectId: GoogleProjectId,
-                                          submissionDate: DateTime,
-                                          terminalStatusDate: Option[DateTime],
-                                          tableName: String,
-                                          datePartitionColumn: Option[String]
-  ): Future[util.List[TableRow]] = {
-
-    val querySql: String =
-      generateSubmissionCostsQuery(submissionId, submissionDate, terminalStatusDate, tableName, datePartitionColumn)
-
-    val namespaceParam =
-      new QueryParameter()
-        .setParameterType(stringParamType)
-        .setParameterValue(new QueryParameterValue().setValue(googleProjectId.value))
-
-    val queryParameters: List[QueryParameter] = List(namespaceParam)
-
-    executeBigQuery(querySql, queryParameters) map { result =>
-      val rowsReturned = Option(result.getTotalRows).getOrElse(0)
-      val bytesProcessed = Option(result.getTotalBytesProcessed).getOrElse(0)
-      logger.debug(
-        s"Queried for costs of submission $submissionId: $rowsReturned Rows Returned and $bytesProcessed Bytes Processed."
-      )
-      Option(result.getRows).getOrElse(List.empty[TableRow].asJava)
-    }
-  }
-
```

Comment:
not strictly part of this PR; this method was totally unused so I got rid of it.